src/api/app/models/project.rb

```diff
@@ -170,6 +170,18 @@ def subprojects
     Project.where("name like ?", "#{name}:%")
   end
 
+  def siblings
+    p = parent
+    siblings = []
```

Comment:
not sure what latest ruby style guides say, but I find it confusing to have a local variable with the same name than the function. But you might be able to fix that by means of my previous comment :)